### ISSUE TITLE
Custom Model Data not showing in menu when called from saved items

### ISSUE THREAD

**Issue by LadyHaley** [elBukkit/MagicPlugin#619]
The following configs produce the following images.

```yml
thanos:
  creator_id: 8d6e627c-1a02-43bf-88b4-bb99b1337400
  creator: LadyHaley
  worth: 0.0
  item:
    ==: org.bukkit.inventory.ItemStack
    v: 1976
    type: GOLD_NUGGET
    meta:
      ==: ItemMeta
      meta-type: UNSPECIFIC
      custom-model-data: 1
```

```yml
appearance_thanos:
  name: "§6Infinity Gauntlet"
  description: "§a6 Stones to Rule Them All"
  upgrade: true
  icon: thanos
  upgrade_icon: thanos
```

```yml
donatorstore:
    parameters:
        auto_close: true
        confirm_filler: stained_glass:3
        show_unavailable: true
        requirements:
        - wand_tags: transformable
        unlock_section: wand_appearances
        apply_to_wand: true
        cost_type: sp
        options:
        - item: none
        - item: none
        - item: none
        - item: appearance_star
          unlock: star
          confirm: true
          cost: 0
        - item: appearance_portal
          unlock: portal
          confirm: true
          cost: 0
        - item: appearance_twilight
          unlock: twilight
          confirm: true
          cost: 0
        - item: appearance_thanos
          unlock: thanos
          confirm: true
          cost: 0
```

In the menu the Item looks like this
![2019-12-16_21-07-57](https://user-images.githubusercontent.com/3754909/71084936-752a6d00-21ea-11ea-9222-fc0914cc3c0a.png)

But looks like this when spawned into my inventory
![2019-12-16_20-54-22](https://user-images.githubusercontent.com/3754909/71084946-7d82a800-21ea-11ea-8728-6e9cc75c2d30.png)

**Comment by NathanWolf** [comment]
Thank you for the detailed bug report!

I have pushed a fix for this, it should work in the latest dev build:

http://jenkins.elmakers.com/job/MagicPlugin/2925/

This will be released soon(ish) as 7.8.1

**Comment by NathanWolf** [comment]
Did this work out for you?
